packages/components/src/flex/test/index.tsx

```diff
@@ -39,6 +44,13 @@ describe( 'props', () => {
 	} );
 
 	test( 'should render align', () => {
+		render(
+			<Flex data-testid="base-flex">
+				<FlexItem>Item</FlexItem>
+				<FlexBlock>Item</FlexBlock>
+			</Flex>
+		);
```

Comment:
Do we really want to render twice here? We probably only want to do the second render, and ignore the one that used to be in `beforeEach`.

There are many other tests like this. For `Flex`, `Scrollable`, `Spacer`, `Surface`, ...

Reply:
We want to because we intend to compare the snapshots of the two renders (see the `toMatchStyleDiffSnapshot ` call below). That applies to the rest of the tests, or at least most of them.

Reply:
Alternatively, we could use [`asFragment()`](https://testing-library.com/docs/react-testing-library/api/#asfragment), but theoretically, it would do the same thing, just with different mechanics.